docs/validator/fullnode.md

```diff
@@ -64,6 +69,19 @@ mv server/data-seed/geth/chaindata node/geth/triecache
 
 :::
 
+:::note
```

Comment:
Genesis sync will be explained in the next part, so better not put it here to keep the structure clear. this part is just for snapshot sync.